internal/runners/packages/search.go

```diff
@@ -153,8 +156,12 @@ func formatSearchResults(packages []*model.IngredientAndVersion) *searchOutput {
 	}
 
 	for i, pack := range packages {
+		name := filterNilStr(pack.Ingredient.Name)
+		if showNamespace {
+			name = fmt.Sprintf("%s/%s", *pack.Ingredient.PrimaryNamespace, name)
+		}
```

Comment:
If you search for a custom namespace it will respond with namespaces as part of the names. This helped me debug, but also felt like a useful enhancement.